src/infinite-list/index.html

```diff
@@ -1,17 +1,17 @@
 <ul ref="wrapper" class="gb-list__wrapper {isGrid ? 'gb-list__wrapper--grid' : 'gb-list__wrapper--column'}">
-  <li data-is="gb-list-item" class="load first-loader" if="{$infinite.isFetchingBackward}" _consumes="infinite">
-    <gb-infinite-loader label="{$infinite.loaderLabel}"></gb-infinite-loader>
+  <li data-is="gb-list-item" class="load first-loader" if="{props.isFetchingBackward}">
+    <gb-infinite-loader label="{props.loaderLabel}"></gb-infinite-loader>
   </li>
-  <li class="load" if="{$infinite.loadMore && $infinite.prevExists}">
-    <gb-button class="load-prev" on-click="{$infinite.clickPrev}">Load Previous</gb-button>
+  <li class="load" if="{props.loadMore && props.prevExists}">
```

Comment:
I know this isn't related to the immediate task, but it seems we made an oversight. Can you please add `data-is="gb-list-item` to this tag? You'll also need to pass down `clickPrev` as a prop to tag.